docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,21 +1,21 @@
 == AeroGear Digger Installation
-https://github.com/aerogear/aerogear-digger-installer[Digger Installer^] - GitHub repo with Ansible playbook used for Digger installation.
+The AeroGear Digger OpenShift project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
 
-The AeroGear Digger project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
+https://github.com/aerogear/aerogear-digger-installer[aerogear-digger-installer^] - GitHub repo with http://docs.ansible.com/ansible/index.html[Ansible^] based automation tool used to install AeroGear Digger.
 
-For more details about Ansible playbooks and variables, please refer to http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs^].
+NOTE: For more details about Ansible playbooks and variables, please refer to http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs^].
 
-It contains the following components:
+NOTE: It is possible to deploy Digger behind HTTP/HTTPS proxy that do not require authentication. see <<using-http-proxy>>
+
+Use the tool to  create an OpenShift project and deploy the following components in it:
 
 * A Jenkins server, used to run the build tasks
-* An http://docs.ansible.com/ansible/index.html[Ansible^] based automation tool used to install and configure components for Android and iOS application building
-* A Docker-formated image with all tools required for Android application building
-* A Nagios server for health monitoring components 
-* A suite of sample Jenkinsfiles used for demonstration of application building using Jenkins pipeline
-* Optional iOS app build functionality by utilising macOS servers with pre-installed required tools. See <<macos-setup>>.
+* A Docker-formatted image with all tools required for Android application building
+* A Nagios server for health monitoring of installed components 
+* Android SDK
```

Comment:
Android-SDK = A Docker-formatted image with all tools required for Android application building

Reply:
resolved

Reply:
update after chat discussion: Android SDK installed to PV for use

Reply:
@laurafitzgerald will providing a list of deployed images help here?

Reply:
resolved(jiraed)